doc/examples/viz_ui.py

```diff
@@ -84,27 +127,47 @@ def modify_button_callback(i_ren, obj, button):
 
 second_button_example.on_left_mouse_button_pressed = modify_button_callback
 
-# /Buttons
+"""
+Panels
+======
 
+Simply create a panel and add elements to it.
+"""
 
-# Panel
 panel = ui.Panel2D(center=(440, 90), size=(300, 150), color=(1, 1, 1), align="right")
 panel.add_element(button_example, 'relative', (0.2, 0.2))
 panel.add_element(second_button_example, 'absolute', (480, 100))
 
-# /Panel
+"""
+TextBox
+=======
+"""
 
-# TextBox
 text = ui.TextBox2D(height=3, width=10)
-# /TextBox
 
-# Show Manager
+"""
+2D Line Slider 
+==============
+"""
+
+line_slider = ui.LineSlider2D(initial_value=-2,
+                              min_value=-5, max_value=5)
+
+"""
+Adding Elements to the ShowManager
+==================================
+
+Once all elements have been initialised, they have 
+to be added to the show manager in the following manner.
+"""
+
 current_size = (600, 600)
 show_manager = window.ShowManager(size=current_size, title="DIPY UI Example")
 
 show_manager.ren.add(cube_actor_1)
 show_manager.ren.add(cube_actor_2)
 show_manager.ren.add(panel)
 show_manager.ren.add(text)
+show_manager.ren.add(line_slider)
 
 show_manager.start()
```

Comment:
We need to comment this. Otherwise the tutorial will block execution of other tutorials when the documentation is generated. 
Also, we need to update viz_advanced.py and viz_widgets.py to use the new line_slider. Basically we need to eradicate widgets from the tutorials.